### DISCUSSION TITLE
An error occurred during modeling, which seems to be related to the amount of data.

### DISCUSSION THREAD

**Discussion by chiyue520** [gempy-project/gempy#774] [Q&A]
Hello!
An error occurred during modeling, which seems to be related to the amount of data.Could you give me any help,please?
Thanks.
![图片](https://user-images.githubusercontent.com/84581378/225302838-f3cb61df-9edd-45cd-a1dd-36878b0c33d9.png)
![Uploading 图片.png…]()

**Comment by javoha** [comment]
How many input points are you using and what is the resolution you are aiming for?
Are you able to provide your dataset?

**Comment by AlexanderJuestel** [comment]
@chiyue520 Please reduce either the number of interface points you use or the resolution of your model
